### PR TITLE
CTOR-851 : Plugin(apps::backup::commvault::commserve::restapi) - Mode(storagepools) : If statement added when $total == 0

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Update documentation

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] 25.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-commvault-commserve-restapi.md [hunk 1]
```diff
@@ -59,108 +106,398 @@ Le connecteur de supervision inclue la supervision des Alerts, Jobs, Media-agent
...
-    --api-password='myapipassword' \
-    --filter-name='IPL' \
-    --warning-space-usage-prct='90' \
-    --critical-space-usage-prct='95' \
-    --verbose
+	--plugin=apps::backup::commvault::commserve::restapi::plugin \
+	--mode=storage-pools \
+	--hostname='10.0.0.1' \
+	--api-username='' \
+	--api-password='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-commvault-commserve-restapi.md:310]: username and password

File: pp/integrations/plugin-packs/procedures/applications-commvault-commserve-restapi.md [hunk 2]
```diff
@@ -61,110 +107,398 @@ To control your Commvault CommServe, the Rest API must be configured.
 
 E.g: https://api.commvault.com/
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-commvault-commserve-restapi.md:108]:
```suggestion
See https://api.commvault.com/
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-commvault-commserve-restapi.md [hunk 3]
```diff
@@ -59,108 +106,398 @@ Le connecteur de supervision inclue la supervision des Alerts, Jobs, Media-agent
 
 Afin de contrôler l'application Commvault CommServe, l'API Rest doit être configuré (cf: https://api.commvault.com/)
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-commvault-commserve-restapi.md:107]:
```suggestion
Afin de superviser l'application Commvault CommServe, l'API Rest doit être configurée (voir https://api.commvault.com/).
```

File: pp/integrations/plugin-packs/procedures/applications-commvault-commserve-restapi.md [hunk 4]
```diff
@@ -61,110 +107,398 @@ To control your Commvault CommServe, the Rest API must be configured.
...
-    --api-password='myapipassword' \
-    --filter-name='IPL' \
-    --warning-space-usage-prct='90' \
-    --critical-space-usage-prct='95' \
-    --verbose
+	--plugin=apps::backup::commvault::commserve::restapi::plugin \
+	--mode=storage-pools \
+	--hostname='10.0.0.1' \
+	--api-username='' \
+	--api-password='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-commvault-commserve-restapi.md:311]: username and password

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-commvault-commserve-restapi.md (hunk 1 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-commvault-commserve-restapi.md:310]:
```suggestion
	--api-username='XXXX' \
	--api-password='XXXX' \
```

File: pp/integrations/plugin-packs/procedures/applications-commvault-commserve-restapi.md (hunk 4 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/applications-commvault-commserve-restapi.md:311]:
```suggestion
	--api-username='XXXX' \
	--api-password='XXXX' \
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**